index.js

```diff
@@ -292,6 +303,46 @@ class Ky {
 
 		return timeout(fetch(this._input, this._options), this._timeout, this.abortController);
 	}
+
+	/* istanbul ignore next */
+	_stream(response, onProgress) {
+		const total = Number(response.headers.get('content-length')) || 0;
+		let transferred = 0;
+
+		return new Response(
+			new ReadableStream({
+				start(controller) {
+					const reader = response.body.getReader();
+
+					if (onProgress) {
+						onProgress(0, 0, total);
+					}
+
+					read();
```

Comment:
It's weird to have this before the definition. Can you move it to below the function?